charts/sourcegraph/values.yaml

```diff
@@ -555,12 +555,8 @@ pgsql:
   # -- Name of existing ConfigMap for `pgsql`. It must contain a `postgresql.conf` key
   existingConfig: "" # Name of an existing configmap
   image:
-    # -- Docker image tag for the `pgsql` image
-    defaultTag: 3.37.0@sha256:af00a4465f435742df9980cc9d8a8bda49d6adea173558a371e276b476ea94ae
-    # -- Docker image name for the `pgsql` image
-    name: "postgres-12.6-alpine"
-  # -- Security context for the `pgsql` container,
-  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)
+    defaultTag: 135107_2022-03-03_9498a8bd3366@sha256:e26b159dc7c0c47d136886390c899816e669a3c2c1ead689bdad0b610364e45e
+    name: "postgres-12-alpine"
```

Comment:
We use the comments like the inline comments to document these values and autogen our docs

```
# -- Docker image tag for the `pgsql` image
```

Would you add them back?